### PR TITLE
Migrate from apiextensions.k8s.io/v1beta1 to apiextensions.k8s.io/v1

### PR DESCRIPTION
**Pull request by TheNavigat**

### CODE REVIEW

File: paasta_tools/setup_kubernetes_internal_crd.py [hunk 1]
```diff
@@ -44,29 +44,33 @@
         },
         spec={
             "group": "paasta.yelp.com",
-            "versions": [{"name": "v1beta1", "served": True, "storage": True}],
+            "versions": [{
+                "name": "v1beta1",
```
Comment (nemacysts) [inline paasta_tools/setup_kubernetes_internal_crd.py:48]: does this version not need to match the api_version above on line 37?

File: paasta_tools/setup_kubernetes_internal_crd.py [hunk 2]
```diff
@@ -76,24 +80,28 @@
...
```
Comment (nemacysts) [inline paasta_tools/setup_kubernetes_internal_crd.py:84]: same as above: does this version not need to match the api_version above on line 73?

**Review by nemacysts — APPROVED** [review]
do we need to feature flag this or is this file to merge regardless of what k8s versions our clusters are running?

**Comment by mattmb** [comment]
Cleaning up and closing some very old PRs. Please re-open or nudge me if you’re still planning to work on this.

**Pull request closed without merging**